docs/paymentMethods/paymentMethodGroups.mdx

```diff
@@ -10,18 +10,31 @@ keywords:
 
 # Payment Method Groups (PMG)
 
-Payment Method Groups offer a way to filter payment methods by a group.
+Payment method groups offer a way to organize and filter payment methods of your users.
+
+It also allows you to display already save payment methods to a user in the checkout form when using
+[Hosted Checkout](../checkouts/hosted-checkout.mdx) or one of the checkout web components (
+[Unified Fintech Checkout](https://storybook.justifi.ai/?path=/docs/payment-facilitation-unified-fintech-checkout%E2%84%A2--docs)
+or [Modular Checkout](https://storybook.justifi.ai/?path=/docs/modular-checkout--docs)).
+
+Here is how to use it:
+
+1. Create a payment method group
+2. Associate existing payment methods
+3. Create a checkout and pass the payment method group ID
 
 ## Creating PMGs
 
-To create a PMG refer to the [Create a Payment Method Group](https://docs.justifi.tech/api-spec#tag/Payment-Method-Groups/operation/CreatePaymentMethodGroup) specification.
+To create a PMG refer to the [Create a Payment Method Group](https://docs.justifi.tech/api-spec#tag/Payment-Method-Groups/operation/CreatePaymentMethodGroup)
+specification.
 
 ## Associating payment methods
 
-There are two ways currently available to associate payment methods to a PMG:
-- **When creating a payment method:**
-Refer to the [Create a Payment Method](https://docs.justifi.tech/api-spec#tag/Payment-Methods/operation/CreatePaymentMethod) specification, an example payload
-is available in the payloads dropdown.
+- **Create Checkout:**
```

Comment:
Oops, I might have put my comment in the wrong place.

Can you please make this section a headline like ""Associating existing payment methods" and move it below "Associating existing payment methods" - so the order and headlines match this part:

Here is how to use it:

1. Create a payment method group
2. Associate existing payment methods
3. Create a checkout and pass the payment method group ID